### PR TITLE
Add image and audio converter classes

### PR DESCRIPTION
**Pull request by mattdangerw**
These classes will occupy the same role as tokenizers for text models. They will transform raw inputs to model inputs in a way that is not task specific.

### CODE REVIEW

**Comment by mattdangerw** [comment]
Still some tests failing here, but I think this is review ready enough. The image converter and audio converter "large" tests will be failing until I add some new assets on Kaggle, but I will wait for a round of review before adding them.

File: keras_nlp/src/utils/preset_utils.py [hunk 1]
```diff
@@ -656,6 +658,14 @@ def load_tokenizer(self, cls, **kwargs):
         """Load a tokenizer layer from the preset."""
         raise NotImplementedError
 
+    def load_audio_converter(self, cls, **kwargs):
+        """Load a audio converter layer from the preset."""
```
Comment (SamanehSaadat) [inline keras_nlp/src/utils/preset_utils.py:662]: an audio

File: keras_nlp/src/utils/preset_utils.py [hunk 2]
```diff
@@ -656,6 +658,14 @@ def load_tokenizer(self, cls, **kwargs):
...
+        raise NotImplementedError
+
+    def load_image_converter(self, cls, **kwargs):
+        """Load a image converter layer from the preset."""
```
Comment (SamanehSaadat) [inline keras_nlp/src/utils/preset_utils.py:666]: an image

**Review by SamanehSaadat — APPROVED** [review]
LGTM! Thanks, Matt!
Just left some nit comments!

**Comment by mattdangerw** [comment]
Assets added on Kaggle. This should be ready to merge now.

**Pull request merged**